drivers/include/at24cxxx.h

```diff
@@ -96,8 +96,22 @@ int at24cxxx_read_byte(const at24cxxx_t *dev, uint32_t pos, void *dest);
  * @return          -ERANGE if @p pos + @p len is out of bounds
  * @return          @see i2c_read_regs
  */
-int at24cxxx_read(const at24cxxx_t *dev, uint32_t pos, void *data,
-                  size_t len);
+int at24cxxx_read(const at24cxxx_t *dev, uint32_t pos, void *data, size_t len);
+
+/**
+ * @brief Sequentially read @p len bytes from a given @p page.
+ *        Similar to @p at24cxxx_read , but this function returns
+ *        the bytes that were read.
```

Comment:
I don't really get why we need the extra function

Reply:
See https://github.com/RIOT-OS/RIOT/pull/19270/files/61027ab724294c33ad517f139afcd8511da2530c#r1106811060